packages/mdc-top-app-bar/README.md

```diff
@@ -38,6 +38,18 @@ npm install @material/top-app-bar
 
 ## Basic Usage
 
+### Menu Icon
```

Comment:
This doesn't belong here. If anything, make it a subheading under HTML Structure.